testsuite/gna/issue1323/testsuite.sh

```diff
@@ -0,0 +1,25 @@
+#! /bin/sh
+
+. ../../testenv.sh
+
+analyze mydesign.vhdl
+elab myentity
+
+simulate myentity --wave=dump.ghw | tee mydesign.out
+
+gcc ../../../src/grt/ghwdump.c ../../../src/grt/ghwlib.c -I../../../src/grt/ -o ghwdump
+
+# We're just checking that ghwdump doesn't crash on a zero length signal.
+./ghwdump -ths dump.ghw > dump.txt
+
+if cmp -s dump.txt golden_dump.txt; then
```

Comment:
```suggestion
if diff dump.txt golden_dump.txt; then
```

This will show any differences in the test log (and continues to show nothing if the test passes).